inspire_dojson/hep/model.py

```diff
@@ -61,6 +61,14 @@ def add_inspire_categories(record, blob):
     return record
 
 
+def copy_special_collections(record, blob):
+    if record.get('_collections') and record.get('special_collections'):
```

Comment:
Also this is incorrect: we need to copy stuff in `_collections` even if the record only has `special_collections`.